packages/fxa-auth-server/lib/geocoding.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  Client,
+  PlaceType2,
+  Status,
+} from '@googlemaps/google-maps-services-js';
+import countries from 'i18n-iso-countries';
+import { ConfigType } from '../config';
+import { Logger } from 'mozlog';
+const config: ConfigType = require('../config').getProperties();
```

Comment:
It would be preferable here to create a class that gets the config via the `Container` API or has it passed in, rather than a module-level import, which also makes it harder to test.

Reply:
Class name suggestion: `GoogleMapsService`

Reply:
I like that. I'll make the necessary change, and rename the file to `googlemapsservice.ts` as well.